docs/sources/alerting/set-up/provision-alerting-resources/index.md

```diff
@@ -0,0 +1,726 @@
+title: Provision Grafana Alerting resources
+weight: 200
+---
+
+# Provision Grafana Alerting resources
+
+Alerting infrastructure is often complex, with many pieces of the pipeline that often live in different places. Scaling this across multiple teams and organizations is an especially challenging task. Grafana Alerting provisioning makes this process easier by enabling you to create, manage, and maintain your alerting data in a way that best suits your organization.
+
+There are three options to choose from:
+
+1. Use file provisioning to provision your Grafana Alerting resources, such as alert rules and contact points, through files on disk.
+
+1. Provision your alerting resources using the Grafana HTTP API.
+
+   For more information on the Grafana Alerting provisioning API, refer to [Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning/).
+
+1. Provision your alerting resources using Terraform.
+
+**Note:**
+
+Currently, provisioning for Grafana Alerting supports alert rules, contact points, mute timings, and templates. Provisioned alerting resources can only be edited in the source that created them and not from within Grafana or any other source. For example, if you provision your alerting resources using files from disk, you cannot edit the data in Terraform or from within Grafana.
+
+**Useful Links:**
+
+[Grafana provisioning](https://grafana.com/docs/grafana/latest/administration/provisioning/)
+
+[Grafana Cloud provisioning](https://grafana.com/docs/grafana-cloud/infrastructure-as-code/terraform/)
+
+[Grafana Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/alerting_provisioning)
+
+## Create and manage alerting resources using file provisioning
+
+Provision your alerting resources using files from disk. When you start Grafana, the data from these files is created in your Grafana system. Grafana adds any new resources you created, updates any that you changed, and deletes old ones.
+
+Arrange your files in a directory in a way that best suits your use case. For example, you can choose a team-based layout where every team has its own file, you can have one big file for all your teams; or you can have one file per resource type.
+
+Details on how to set up the files and which fields are required for each object are listed below depending on which resource you are provisioning.
+
+**Note:**
+
+Provisioning takes place during the initial set up of your Grafana system, but you can re-run it at any time using the [Grafana Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations).
+
+### Provision alert rules
+
+Create or delete alert rules in your Grafana instance(s).
+
+1. Create an alert rule in Grafana.
+1. Use the [Alerting provisioning API](https://grafana.com/docs/grafana/latest/developers/http_api/admin/#reload-provisioning-configurations) to extract the alert rule.
```

Comment:
IMHO this is gonna be done by everyone thus we should provide more details regarding the call to the API here.

Reply:
@JohnnyQQQQ can you help me here, please?